README.md

```diff
@@ -6,3 +6,11 @@
 2. Update contract details in the `config` folder
 3. Authenticate with `yarn graph auth https://api.thegraph.com/deploy/ <access-token>`
 4. Deploy with `yarn deploy:<network>`
+
+Note: For the Base network subgraph (which is deployed to Alchemy), step 3 above should be skipped, and it will be necessary to create a .env file with the following content:
+
+```
+BASE_DEPLOY_KEY=<access-token>
```

Comment:
Hi John, let's add an `.env.example`, so it's easier to spot!

Reply:
Good idea.. done: https://github.com/ScopeLift/umbra-subgraph/pull/6/commits/e2d5442254b908fd2d42ad8597b36df894211a15